### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by spacedmonkey**
Trac ticket: https://core.trac.wordpress.org/ticket/57814

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: tests/phpunit/tests/webfonts/wpThemeJsonWebfontsHandler.php [hunk 1]
```diff
@@ -131,6 +131,7 @@ private function setup_theme_and_test( $theme_name ) {
 		switch_theme( $theme_name );
 		do_action( 'after_setup_theme' );
 		wp_clean_theme_json_cache();
+		do_action( 'plugins_loaded' );
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/webfonts/wpThemeJsonWebfontsHandler.php:134]: Is this necessary as we are checking the theme JSON?

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @spacedmonkey for quick PR. Left one question other then this LGTM.

File: tests/phpunit/tests/webfonts/wpThemeJsonWebfontsHandler.php (hunk 1 above)
Comment (spacedmonkey) [reply tests/phpunit/tests/webfonts/wpThemeJsonWebfontsHandler.php:134]: @mukeshpanchal27 https://github.com/WordPress/wordpress-develop/blob/39bfc2580d9b0ea7e6ff45b8d904408d925cbc4b/src/wp-includes/default-filters.php#L354

**Comment by spacedmonkey** [comment]
Committed

**Pull request closed without merging**